modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -344,15 +517,15 @@ The `linenumber()` function provides exactly that or `null` if called without a
 .Query
 [source, cypher]
 ----
-LOAD CSV FROM 'file:///artists.csv' AS line
-RETURN linenumber() AS number, line
+LOAD CSV FROM 'file:///artists.csv' AS row
+RETURN linenumber() AS number, row
 ----
 
 .Result
-[role="queryresult"]
+[source, role="queryresult"]
 ----
 +---------------------------------------+
```

Comment:
We tend to create tables for the Result columns in Cypher Manual (something I am beginning to reconsider), but for now, maybe we can stick to how other pages do it and then consider a more holistic change later. 

```suggestion
[role="queryresult",options="header,footer",cols="2m"]
|===
| number | row

| 1 | ["1","ABBA","1992"]
| 2 | ["2","Roxette","1986"]
| 3 | ["3","Europe","1979"]
| 4 | ["4","The Cardigans","1992"]

2+d|Rows: 4
|===
```